### DISCUSSION TITLE
wire:model seem strange

### DISCUSSION THREAD

**Discussion by tegarmonster** [livewire/livewire#8705] [1. Help]
Hi,
I wonder why, why form value not getting the newest value when move to another page, I mean the step is like this:
1. update form value in page A,
2. success update,
3. move to page B,
4. back again to page A
and the form value still show the old value before update

I'm using wire:model to binding the value

**Comment by EdgarsJoja** [comment]
Once you move to a different page, the previous state is gone. If you want to store & restore it, you have to keep those changes somewhere, e.g. session. So that when you go back you can initiate your values from the session data.

Check this in docs: https://livewire.laravel.com/docs/session-properties
